client/app/bundles/shared/components/Logo.jsx

```diff
@@ -7,14 +7,15 @@ type Props = {
   size?: string; // Future Task: Use ScreenSize enum
 }
 
+// eslint-disable-next-line react/prefer-stateless-function
 export default class Logo extends React.Component<Props, {}> {
   render() {
     const { onClick, size = '' } = this.props;
     const linkClass = onClick ? 'link' : '';
     const containerClass = `${css.container} ${css[size] || ''} ${linkClass}`;
 
     return (
-      <div className={containerClass} onClick={onClick}>
+      <div role={'presentation'} className={containerClass} onClick={onClick}>
```

Comment:
Can we just do `role="presentation"`?

Reply:
Can modify the eslint rules; it currently set to enforce all values to be wrapped ｛｝

Reply:
Oh that's fine then! I was just wondering! It seemed a little redundant to pass a string into a {}.